samples/streaming/provider/src/provider_impl.rs

```diff
@@ -77,7 +75,12 @@ impl Iterator for ImageFileIterator {
         // Note: We will go back to the start of the list once we past the end.
         self.current_image_file_index = (self.current_image_file_index + 1) % len;
 
-        Some(StreamResponse { content: self.read_image_file(current_image_filename).ok()? })
+        let media = Media {
+            media_type: mime_type::JPEG_IMAGES.to_string(),
```

Comment:
should we do some kind of verification that it's actually a jpeg? as far as I can tell the `read_image_file` is just getting raw bytes, so this is potentially not the right mime type. maybe just check that the file extension is jpg or jpeg

Reply:
It's a sample.  Not worried.